plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/status/StatusCommandPresenter.java

```diff
@@ -89,27 +93,28 @@ protected void onFailure(Throwable exception) {
      *
      * @param statusText
      *         text to be printed
-     * @param console
-     *         console for displaying status
      */
-    private void printGitStatus(String statusText, GitOutputConsole console) {
-
+    private void printGitStatus(String statusText) {
+        GitOutputConsole console = gitOutputConsoleFactory.create(STATUS_COMMAND_NAME);
         console.print("");
```

Comment:
It is not my changes, but I think it is needed to separate title line from status text

Project name: test
 
On branch master
Your branch and 'origin/master' have diverged,
and have 1 and 1 different commit each, respectively.
  (use "git pull" to merge the remote branch into yours)
Changes to be committed:
  (use "git reset HEAD <file>..." to unstage)
 
	deleted:    README.md
	new file:   READMdE.md
 
Untracked files:
  (use "git add <file>..." to include in what will be committed)
 
	rfhsehr